### PR TITLE
UIDEXP-32: Add Save icon to the Save Instances UUID

### PR DESCRIPTION
**Pull request by Yuliia-Dovhal**
## Purpose:
Add Save icon to the Save Instances UUIDs option in Action menu

[UIDEXP-32](https://issues.folio.org/browse/UIDEXP-32)

## Screenshot:
Screenshot 2020-03-10 at 11 17 40

### CODE REVIEW

File: src/components/InstancesList/InstancesList.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ import {
 
 import { AppIcon } from '@folio/stripes/core';
 import { SearchAndSort } from '@folio/stripes/smart-components';
-import { Button } from '@folio/stripes/components';
+import { Button, Icon } from '@folio/stripes/components';
```
Comment (YevheniiMaltsev) [inline src/components/InstancesList/InstancesList.js]:
```suggestion
import {
  Button,
  Icon,
} from '@folio/stripes/components';
```

File: src/components/InstancesList/InstancesList.js [hunk 2]
```diff
@@ -193,6 +195,11 @@ class InstancesView extends React.Component {
             this.generateInstancesIdReport();
           }}
         >
+          <Icon
```
Comment (YevheniiMaltsev) [inline src/components/InstancesList/InstancesList.js]: I think we can try to add a test to check if the correct (save) icon is in place. @VictorSoroka1  @Yuliia-Dovhal  what do you think about it?

File: src/components/InstancesList/InstancesList.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ import {
 
 import { AppIcon } from '@folio/stripes/core';
```
Comment (YevheniiMaltsev) [inline src/components/InstancesList/InstancesList.js]: Please, add a screenshot to the pull request description

File: src/components/InstancesList/InstancesList.js (hunk 2 above)
Comment (Yuliia-Dovhal) [reply src/components/InstancesList/InstancesList.js]: The existed tests are broken and I will spend too much time for fixing all that tests before even adding my ones. Also it seems like fixing existing tests is not only my responsibility.
Maybe @VictorSoroka1 can disagree?

Comment (VictorSoroka1) [reply src/components/InstancesList/InstancesList.js]: Agree.

**Pull request merged**